data.sql

```diff
@@ -0,0 +1,33 @@
+/* Populate database with sample data. */
+
+INSERT INTO animals VALUES(1,'Agumon','Feb 03, 2020',0,true,10.23);
+
+INSERT INTO animals VALUES(2,'Gabumon','Nov 15, 2018',2,true,8);
+
+INSERT INTO animals VALUES(3,'Pikachu','Jan 07, 2021',1,false,15.04);
+
+INSERT INTO animals VALUES(4,'Devimon','May 12, 2017',5,true,11);
+
+INSERT INTO animals VALUES(5,'Charmander', 'feb 08,2020',0,false,-11);
+
+INSERT INTO animals VALUES(6,'Plantmon', 'nov 15,2021',2,true,-5.7);
+
+INSERT INTO animals VALUES(7,'Squirtle', 'apr 02,1993',3,false,-12.13);
+
+INSERT INTO animals VALUES(8,'Angemon', 'jul 07,2005',1,true,-45);
+
+INSERT INTO animals VALUES(9,'Boarmon', 'jul 07,2005',7,true,20.14);
+
+INSERT INTO animals VALUES(10,'Blossom', 'oct 13,1998',3,true,17);
+
+INSERT INTO animals VALUES(11,'Ditto', 'may 14,2022',4,true,22,);
+
+INSERT INTO  owners (full_name,age) VALUES
+('Sam Smith', 34),
+('Jennifer Orwell',19),
+('Bob',45),
+('Melody Pond',77),
+('Dean Winchester',14),
+('Jodie Whittaker',38);
+
+INSERT INTO species (name) VALUES ('Pokemon'),('Digimon');
```

Comment:
- [ ] they should be a query where if an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead something like this
```
UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Digimon')
   WHERE name like '%mon';
```
- [ ] There should also be a query where all animals got assigned owners
```
UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';
```